docs(contributor): add missing link to mxGeometry source

tbouffard opened the pull request:
The link had been removed by mistake while migrating the content from AsciiDoc to Markdown long time ago (November 2020).

### Notes

Detected in #2366
Mistake done in #892

csouchet reviewed (changes requested):
I don't know if the problem comes from me, but the link doesn't work :slightly\_frowning\_face:   
  
 

![](https://user-images.githubusercontent.com/4921914/220858718-7d21b4e0-dcbb-4269-b7a7-37826ba8d786.png)

tbouffard commented:
> I don't know if the problem comes from me, but the link doesn't work slightly_frowning_face 

My bad, I didn't check the link. 👿 
I added the link previously used in the AsciiDoc file. In the meantime, the `mxgraph2` repository has been removed or made private, so the target URL generates a HTTP 404.
This is now fixed in 05fbfaa

csouchet reviewed (approved):
Works fine now :smile: 

![](https://user-images.githubusercontent.com/4921914/220862094-68ddc306-acfc-456f-bad8-fdf56f760893.png)
